src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -2069,6 +2069,13 @@ TEST_CASE("temp entry eviction", "[tx][soroban]")
         closeLedgerOn(*test.getApp(), i, 2, 1, 2016);
     }
 
+    // This should be a noop
+    test.extendOp({lk}, 10'000);
```

Comment:
We should add a TTL check afterwards to make sure this was actually a n-op, perhaps after the host function call as well just to be safe.

Reply:
Is the TTL check on line 2081 not sufficient?

Reply:
Yeah it's probably fine as is. A TTL check is slightly stronger, since a buggy extend op could change the TTL but still have the entry be expired. The `isEntryLive` check wouldn't pick this up but the TTL check would. If something is supposed to be a no-op I also like checking immediately following the operation that is had no side effects. Checking that two ops result in a no-op is different than checking that each op individually is a no-op, but in practice I don't think it matters too much, I'm probably being a bit pedantic...

Reply:
A that's a good point. Updated.